web/src/features/fbaCalculator/components/HFICell.tsx

```diff
@@ -0,0 +1,68 @@
+import { makeStyles, TableCell } from '@material-ui/core'
+import { AnyRecord } from 'dns'
+import React, { CSSProperties } from 'react'
+import FixedDecimalNumberCell from './FixedDecimalNumberCell'
+
+interface HFICellProps {
+  value: number | undefined
+  className?: string
+}
+
+const useStyles = makeStyles({
+  dataRow: {
+    height: '40px',
+    paddingLeft: '8px',
+    paddingRight: '8px'
+  },
+  orangeBorder: {
+    border: 'solid 3px #FFC464',
+    height: '40px',
+    paddingLeft: '8px',
+    paddingRight: '8px'
+  },
+  orangeFill: {
+    backgroundColor: '#FFC464',
+    height: '40px',
+    paddingLeft: '8px',
+    paddingRight: '8px'
+  },
+  redFill: {
+    backgroundColor: '#FF6259',
+    height: '40px',
+    paddingLeft: '8px',
+    paddingRight: '8px'
+  },
+})
+
+
+
+const DECIMAL_PLACES = 1
+
+const HFICell = (props: HFICellProps) => {
+  const classes = useStyles()
+
+  const HFIStyle = getHFIStyle(props.value);
+
+  function getHFIStyle(value:number | undefined): string  {
```

Comment:
Minor: this can be an arrow function: https://basarat.gitbook.io/typescript/future-javascript/arrow-functions

Reply:
Changed